server/start.go

```diff
@@ -289,6 +289,10 @@ func startInProcess(ctx *Context, clientCtx client.Context, appCreator types.App
 
 		app.RegisterTxService(clientCtx)
 		app.RegisterTendermintService(clientCtx)
+
+		if a, ok := app.(types.ApplicationQueryService); ok {
+			a.RegisterNodeService(clientCtx)
+		}
```

Comment:
whats a possible condition that app might not be implementing `ApplicationQueryService`?

Reply:
I'm assuming it has something related to the versionings of sdk 0.46, and trying not to break it, but don't have a clear idea in my mind

Reply:
Yeah that's correct. It's to avoid having SDK 0.46 app breaking our app API